### PR TITLE
Exchange code to be compatible to Version 11.0 and exchange screenshots.

### PR DESCRIPTION
**Pull request by linawolf**

### CODE REVIEW

File: Documentation/4-FirstExtension/5-controlling-the-flow.rst [hunk 1]
```diff
@@ -34,22 +35,31 @@ In our simple example, the controller looks like this:
...
+         * @param \T3docs\StoreInventory\Domain\Repository\ProductRepository $productRepository
          */
         public function injectProductRepository(ProductRepository $productRepository)
         {
             $this->productRepository = $productRepository;
         }
 
-        public function listAction()
+        /**
+        * List Action
```
Comment (alexanderschnitzler) [inline Documentation/4-FirstExtension/5-controlling-the-flow.rst:46]: There is no proper indentation here. But besides that, this doc block doesn't add any information. The native return type is quite sufficient in my opinion.

File: Documentation/4-FirstExtension/7-configuring-the-plugin.rst [hunk 2]
```diff
@@ -17,20 +17,24 @@ our extension directory.
...
-        // non-cacheable actions
-        [
-            \Vendor\StoreInventory\Controller\StoreInventoryController::class => '',
-        ]
-    );
+   <?php
+   // Prevent script from being called directly
+   defined('TYPO3') or die();
+
+   // encapsulate all locally defined variables
```
Comment (alexanderschnitzler) [inline Documentation/4-FirstExtension/7-configuring-the-plugin.rst:24]: The problem is, there are no local variables. Hence, this example is non explanatory.

Comment (linawolf) [reply Documentation/4-FirstExtension/7-configuring-the-plugin.rst:24]: I took the example from here: https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/ExtensionArchitecture/ConfigurationFiles/Index.html?highlight=ext_localconf%20php#examples
This seems to be new best practise...

**Comment by linawolf** [comment]
Merging this now, please make another PR if you still want to change the code example @alexanderschnitzler

**Pull request merged**